src/lib/utils.ts

```diff
@@ -91,3 +91,18 @@ export const isMetaNode = (nodeType: string) =>
   nodeType === schema.nodes.bibliography_item.name ||
   nodeType === schema.nodes.affiliation.name ||
   nodeType === schema.nodes.contributor.name
+
+export const createFootnoteLabel = (id: string) => {
```

Comment:
We have a `footnotes-utils` file that could be a good place for this.

Reply:
moved from utils to footnotes-utils